components/admin/data/layers/table/constants.js

```diff
@@ -0,0 +1,14 @@
+export const INITIAL_PAGINATION = {
+  // displays pagination
+  enabled: true,
+  // number of items per page
+  limit: 20,
+  // current page
+  page: 1,
+  // total of pages
+  pages: null,
+  // total of items to display
+  size: null
+};
+
+export default { INITIAL_PAGINATION };
```

Comment:
This file is the same for the three tabs (layers, widgets and datasets), maybe we should move things around and let them share it

Reply:
Let's not share states. This could give us problems in the future. Keep them as independent local states.